Sample/AppDelegate.swift

```diff
@@ -25,9 +25,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
   func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey : Any]? = nil) -> Bool {
     ASDisableLogging()
-    if ProcessInfo.processInfo.arguments.contains("UITests") {
-      RRFPSBar.sharedInstance().isHidden = false
-    }
+    RRFPSBar.sharedInstance().isHidden = ProcessInfo.processInfo.arguments.contains("UITests")
```

Comment:
One more annoying comment: this will cause the shared instance to be created in all cases, even if were just hiding it. Better to keep it inside an `if` and just fix the condition, I think.